__tests__/api.test.js

```diff
@@ -123,14 +123,54 @@ describe("GET /api/articles", () => {
         expect(articles.length).toEqual(13);
       });
   });
-  test("should be ordered by date descending", () => {
+  test("should be ordered by date descending default", () => {
     return request(app)
       .get("/api/articles")
       .then(({ body }) => {
         const { articles } = body;
         expect(articles).toBeSortedBy("created_at", { descending: true });
       });
   });
+  test("should be filtered by topic mitch", () => {
+    return request(app)
+      .get("/api/articles?topic=mitch")
```

Comment:
Ensure you're expecting a 200 status for each of these tests with a valid query, you've missed a few in here 👍